openstack/vpn/v5/connections/results.go

```diff
@@ -0,0 +1,106 @@
+package connections
+
+import "github.com/chnsz/golangsdk/pagination"
+
+type Connections struct {
+	ID                  string       `json:"id"`
+	Name                string       `json:"name"`
+	Status              string       `json:"status"`
+	VgwId               string       `json:"vgw_id"`
+	VgwIp               string       `json:"vgw_ip"`
+	Style               string       `json:"style"`
+	CgwId               string       `json:"cgw_id"`
+	PeerSubnets         []string     `json:"peer_subnets"`
+	TunnelLocalAddress  string       `json:"tunnel_local_address"`
+	TunnelPeerAddress   string       `json:"tunnel_peer_address"`
+	EnableNqa           bool         `json:"enable_nqa"`
+	PolicyRules         []PolicyRule `json:"policy_rules"`
+	Ikepolicy           IkePolicy    `json:"ikepolicy"`
+	Ipsecpolicy         IpsecPolicy  `json:"ipsecpolicy"`
+	CreatedAt           string       `json:"created_at"`
+	UpdatedAt           string       `json:"updated_at"`
+	EnterpriseProjectID string       `json:"enterprise_project_id"`
+	ConnectionMonitorID string       `json:"connection_monitor_id"`
+	HaRole              string       `json:"ha_role"`
+}
+
+type listResp struct {
+	Connections []Connections `json:"vpn_connections"`
+	RequestId   string        `json:"request_id"`
+	PageInfo    pageInfo      `json:"page_info"`
+	TotalCount  int64         `json:"total_count"`
+}
+
+type PolicyRule struct {
+	RuleIndex   int8     `json:"rule_index"`
```

Comment:
just use `int` for Integer and `int64` for Long, don't use other types

Reply:
down